packages/material-ui-lab/src/themeAugmentation/components.d.ts

```diff
@@ -0,0 +1,128 @@
+import { ComponentsProps, Overrides, Variants } from '@material-ui/core';
+
+export interface LabComponents {
+  MuiAlert?: {
+    props?: ComponentsProps['MuiAlert'];
+    variants?: Variants['MuiAlert'];
+    overrides?: Overrides['MuiAlert'];
```

Comment:
I wonder about the naming convention. It could be

```suggestion
    props?: Props['MuiAlert'];
    variants?: Variants['MuiAlert'];
    overrides?: Overrides['MuiAlert'];
```

or 

```suggestion
    props?: ComponentsProps['MuiAlert'];
    variants?: ComponentsVariants['MuiAlert'];
    overrides?: ComponentsOverrides['MuiAlert'];
```

Maybe the latter?

Reply:
Also, I was expecting relative imports, it seems strange to have these values coming from the core and not the lab.

Reply:
We are doing module augmentation and the variants are just inherited from the ComponentProps I think, so everything works. But I agree it looks confusing, let me see how can it be defined better.

Reply:
Adding Components prefix makes sense 👍